homeassistant/components/mqtt/__init__.py

```diff
@@ -145,6 +145,7 @@
     "fan",
     "light",
     "lock",
+    "number",
```

Comment:
The MQTT integration doesn't support the number platform. What does the change here do?

Reply:
Sorry forgot to commit the most important bit 😅